Tests/TuistGeneratorTests/ProjectMappers/AutogeneratedSchemesProjectMapperTests.swift

```diff
@@ -81,6 +93,16 @@ final class AutogeneratedSchemesProjectMapperTests: TuistUnitTestCase {
                     projectPath: projectPath,
                     testTargetName: "BTests"
                 ),
+                testScheme(
+                    target: targetBTests,
+                    projectPath: projectPath,
+                    testTargetName: "BUITests"
+                ),
+                testScheme(
+                    target: targetBTests,
+                    projectPath: projectPath,
+                    testTargetName: "BIntegrationTests"
+                ),
```

Comment:
I believe we'll need to update the test helper `testScheme` to allow specifying an array of test target names to cater for the new changes - as in this test case `B` target will have 3 test targets in its scheme.

e.g.

```swift
XCTAssertEqual(
            got.schemes,
            [
                testScheme(
                    target: targetA,
                    projectPath: projectPath,
                    testTargetNames: ["ATests"]
                ),
                testScheme(
                    target: targetATests,
                    projectPath: projectPath,
                    testTargetNames: ["ATests"]
                ),
                testScheme(
                    target: targetB,
                    projectPath: projectPath,
                    testTargetNames: ["BIntegrationTests", "BTests", "BUITests"]
                ),
                testScheme(
                    target: targetBTests,
                    projectPath: projectPath,
                    testTargetNames: ["BTests"]
                ),
                testScheme(
                    target: targetBUITests,
                    projectPath: projectPath,
                    testTargetNames: ["BUITests"]
                ),
                testScheme(
                    target: targetBIntegrationTests,
                    projectPath: projectPath,
                    testTargetNames: ["BIntegrationTests"]
                ),
            ]
        )
```

_(heh re-examing `testScheme` I can see how it can be confusing in this context 😅, perhaps a better choice would have been `makeScheme`)_